swift_browser_ui_frontend/src/entries/main.js

```diff
@@ -309,27 +309,19 @@ new Vue({
       return active;
     };
     initialize().then(ret => {
-      if (this.$te("message.keys")) {
-        for (let item of Object.entries(this.$t("message.keys"))) {
-          let keyURL = new URL(
-            "/download/" +
-              item[1]["project"] +
-              "/" +
-              item[1]["container"] +
-              "/" +
-              item[1]["object"],
-            document.location.origin,
-          );
-          keyURL.searchParams.append("project", ret.id);
-          fetch(keyURL)
-            .then(resp => {
-              return resp.text();
-            })
-            .then(resp => {
-              this.$store.commit("appendPubKey", resp);
-            });
-        }
-      }
+      let keyURL = new URL(
```

Comment:
This URL doesn't match the path for getting a public key from the upload runner, thus not resolving to anything but 404. I already wrote a fix for it in the decrypted download implementation, I can open a PR about it on this branch if it's fine by you?

Reply:
Sure, just push to this branch